lxc/image.go

```diff
@@ -55,11 +55,16 @@ func (a SortImage) Less(i, j int) bool {
 }
 
 type aliasList []string
+type algorithm []string
```

Comment:
This whole set of change to lxc/image.go is un-needed. The algorithm should just be string property of publishCmd in lxc/publish.go